jobs/scripts/gluster-block/lcov.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+yum install -y git autoconf automake gcc libtool make file rpcgen glusterfs-api-devel libuuid-devel json-c-devel libtirpc-devel
+
+# install runtime dependencies
+yum install -y glusterfs-server targetcli tcmu-runner
+
+# get the gluster-block source code
+rm -rf gluster-block/
+git clone --depth 2 https://github.com/gluster/gluster-block/
+
+nproc=$(getconf _NPROCESSORS_ONLN)
+
+# compile and istall from source code with lcov
+pushd gluster-block
+./autogen.sh
+./configure CFLAGS="-g3 -O0 -lgcov --coverage -fprofile-arcs -ftest-coverage"
+make install -j ${nproc}
+
+echo "Initializing the line coverage"
+mkdir coverage
+lcov -d . --zerocounters
+lcov -i -c -d . -o coverage/gluster-block-lcov.info
+set +e
+
+# start glusterd process
+systemctl start glusterd
+
+# run the test cases
+./tests/basic.t
```

Comment:
thinking more about it... how about implementing a 'script' which runs all the tests, and calling that here, so we needn't make changes to this file always, and can keep adding .t files into gluster-block project. Not a merge blocker for this patch, but once it gets running, we need to do this as a followup step.

Reply:
I totally agree.